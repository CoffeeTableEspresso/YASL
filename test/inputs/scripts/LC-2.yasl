
/*
 * Taken from: https://leetcode.com/problems/defanging-an-ip-address/
 *
 * Given a valid (IPv4) IP address, return a defanged version of that IP address.
 * A defanged IP address replaces every period "." with "[.]".
 */

fn f(const ip) {
    return ip->replace('.', '[.]')
}

echo f('255.100.50.0')
echo f('1.1.1.1')
