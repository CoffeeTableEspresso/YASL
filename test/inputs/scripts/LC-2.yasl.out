255[.]100[.]50[.]0
1[.]1[.]1[.]1
